app/upgrades/v22/upgrades.go

```diff
@@ -23,6 +25,13 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		// Properly register consensus params. In the process, change params as per:
+		// https://forum.osmosis.zone/t/raise-maximum-gas-to-300m-and-lower-max-bytes-to-5mb/1116
+		defaultConsensusParams := tmtypes.DefaultConsensusParams().ToProto()
+		defaultConsensusParams.Block.MaxBytes = 5000000
+		defaultConsensusParams.Block.MaxGas = 300000000
```

Comment:
just for context, what's the reason we're only dealing with max byte and gas here + how did we get these numbers?

Reply:
We utilize all defaults, except for the two block params that the in line comment links to.

Reply:
Can the code comment paste what mainnet used to have>

Reply:
Added previous mainnet values in-line comments here https://github.com/osmosis-labs/osmosis/pull/7220/commits/cb054741ed023818ddc8e0d98a3333a293c0b998